wicketd/src/artifacts.rs

```diff
@@ -806,22 +868,61 @@ mod tests {
         RandomHostOsImage { phase1, phase2, tarball }
     }
 
+    struct RandomRotImage {
+        archive_a: Bytes,
+        archive_b: Bytes,
+        tarball: Bytes,
+    }
+
+    fn make_random_rot_image() -> RandomRotImage {
```

Comment:
I feel like this can potentially be unified with the rot image generator in tufaceous-lib, though I definitely don't want to block on this. (Might have to refactor the code in tufaceous-lib a little.)

Reply:
Yeah, definitely agreed. I have a tentative plan to make `tufaceous-lib` a little more knowledgeable about RoT and host composite artifacts, which would let us clean up both this and `make_random_host_image()` on which this is based.